Backend/lib/yudhishthira/src/Lib/Yudhishthira/Types.hs

```diff
@@ -119,13 +119,15 @@ data LogicDomain
   = POOLING
   | FARE_POLICY
   | DYNAMIC_PRICING DST.ServiceTierType
+  | CONFIG
```

Comment:
Is this applicable to all configurations? If not, should we rename it to be specific to the configuration in question?

Reply:
i was also thinking of doing in that way but we can make it generic also so put it this way i will make it driver_pool_config

Reply:
or we can have `| CONFIG Text (Config configName)` then it can be same tag can be reused.